### ISSUE TITLE
enable/disable-service-access does not account situations where `org` is not used

### ISSUE THREAD

**Issue by aegershman** [nulldriver/cf-cli-resource#41]
[The enable/disable-service-access commands should work on one or all orgs](https://github.com/nulldriver/cf-cli-resource#enable-service-access). To enable/disable service access for _all_ orgs you do not provide any `org` as a param. You use `cf enable-service-access SERVICE` and nothing else.

However, if you do not provide the resource an `org` name (which would be appropriate if you wanted to affect *all* orgs), you get an error [at line 20 of `cf-functions.sh`, which handles the login logic](https://github.com/nulldriver/cf-cli-resource/blob/master/assets/cf-functions.sh#L20)

[The last version which I have validated works is `2.4.0`](https://github.com/nulldriver/cf-cli-resource/blob/v2.4.0/assets/cf-functions.sh#L19)

Examples:

```
# disable-access results in...
[INFO] Executing disable-service-access: p-redis
/opt/resource/cf-functions.sh: line 20: 1: org null or not set

# enable-access results in...
[INFO] Executing enable-service-access: p-rabbitmq
/opt/resource/cf-functions.sh: line 20: 1: org null or not set
```

minimal example pipeline:

```yml
resource_types:
- name: cf-cli-resource
  type: docker-image
  source:
    repository: nulldriver/cf-cli-resource
    tag: 2.8.2

resources:
- name: cf-env
  type: cf-cli-resource
  source:
    api: myapi
    username: myusername
    password: mypassword
    skip_cert_check: true

jobs:
- name: enforce-service-access-state
  plan:
    - put: p-redis
      resource: cf-env
      params: 
        command: disable-service-access
        service_broker: p-redis
    - put: p-rabbitmq
      resource: cf-env
      params: 
        command: enable-service-access
        service_broker: p-rabbitmq
```

**Comment by aegershman** [comment]
[I noticed you released v2.8.3](https://github.com/nulldriver/cf-cli-resource/commit/045c3dd118eae07b1504f67d9aac33d81cbc697f), but it doesn't appear as though that release is available on the [releases page of this repo](https://github.com/nulldriver/cf-cli-resource/releases) or as a [released version on dockerhub](https://hub.docker.com/r/nulldriver/cf-cli-resource/tags/)

No rush, no worries, etc. etc., but just pointing it out just in case it wasn't intentional

Thanks a ton for your time and contributions

**Comment by patrickcrocker** [comment]
The CI job is failing so it hasn’t published yet. Will take a look once my
day job allows me some time 😁

**Comment by patrickcrocker** [comment]
The CI job is now fixed (stupid race condition...)

This is now fixed in [v2.8.3](https://github.com/nulldriver/cf-cli-resource/releases/tag/v2.8.3)

**Comment by aegershman** [comment]
Sick 🤙 Thank you.
